Dockerfile

```diff
@@ -1,18 +1,19 @@
 # Basic Dockerfile for the USASpendingAPI
 
-## 0) Add your DATABASE_URL on the ENV line below
-## 1) Build:
+## 0) Add your DATABASE_URL on the ENV line below. Use host.docker.internal instead of localhost.
+## 1) Run server w/docker-compose, check port 8000:
+#        docker-compose up --build
+## Optional) Run ad-hoc commands:
 #        docker build . -t usaspendingapi
-## 2) Run (include -p flag and ports)
-#        docker run -p 127.0.0.1:8000:8000 usaspendingapi
+#        docker run -p 127.0.0.1:8000:8000 usaspendingapi <command>
 
-# This will forward port 8000 of the container to your localhost:8000 and start a new container for the API.
 # Rebuild and run when code in /usaspending-api changes
 
 FROM python:3.5.3
```

Comment:
Recommend dropping the third version number to have python:3.5. 
Future work could be to use centos as the base image